newm-shared/src/main/kotlin/io/newm/shared/exposed/ExposedArrays.kt

```diff
@@ -1,6 +1,7 @@
 package io.newm.shared.exposed
```

Comment:
Changes on this file are to be able to support nullable array columns (previously we only supported non-nullable)